.github/workflows/create-release.yaml

```diff
@@ -0,0 +1,24 @@
+name: "Create release"
+
+on:
+  workflow_dispatch:
+    inputs:
+      name:
+        description: "Create release"
+        default: ""
+        required: true
+      since:
+        description: "Changelog since"
+        default: ""
+        required: false
+
+jobs:
+  create-draft:
+    name: Create draft release
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Create lightweight tag
```

Comment:
Shouldn't we checkout before operating on the repository?

Reply:
I am not absolutely sure it is necessary, so I added checkout.